src/libs/API.js

```diff
@@ -496,7 +496,7 @@ function Report_GetHistory(parameters) {
     const commandName = 'Report_GetHistory';
     requireParameters(['reportID'],
         parameters, commandName);
-    return request(commandName, parameters);
+    return request(commandName, {...parameters, reportActionsLimit: CONST.REPORT.REPORT_ACTIONS_LIMIT});
```

Comment:
I think we're not supposed to use the spread operator

Reply:
Nah, I think the spread operator is fine here. Just have to be careful w/ it because the order matters.

```javascript
const myObject = {
    a: 1,
    b: 2,
};

console.log({...myObject, {a: 'override'}});  // This will be {a: 'override', b: 2}
console.log({{a: 'default'}, ...myObject},);    // This will be {a: 1, b: 2}
```

Reply:
I thought you would just do `parameters['reportID'], parameters['reportActionsOffset'], ...` 
But I'm not 100% sure, I know if seen that linting error about the spread operator. Maybe it is just for props.

Reply:
Sounds good :+1: